[ADD] Player entity

adikmain opened the pull request:
[Change] application.properties password for postgres

adikmain committed:
[ADD] Player entity
[Change] application.properties password for postgres

Orynbasar commented on src/main/resources/application.properties:
```diff
@@ -1,4 +1,4 @@
 spring.datasource.url=${DB_URL:jdbc:postgresql://localhost:5432/fb}
 spring.datasource.username=${DB_USERNAME:postgres}
-spring.datasource.password=${DB_PASSWORD:root}
+spring.datasource.password=${DB_PASSWORD:Adiqq228}
```
держи это у себя локально. Не надо комитить такое

Orynbasar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,27 @@
+@Entity
+public class Player {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(
+            name = "UUID",
+            strategy = "org.hibernate.id.UUIDGenerator"
+    )
```
Убери это, @GeneratedValue(strategy = GenerationType.AUTO) 
достаточно. У нас на стороне БД будут генериться ID

Orynbasar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,27 @@
+    @Column(name = "steam_id", updatable = true, nullable = false)
+    private String steam_id;
+    @Column(name = "simple_mmr", updatable = true, nullable = false)
+    private int simple_mmr;
```
camel case: simpleMmr

adikmain committed:
[FIX] Pull request comments

application.properties password
Player entity's variable namesz

onggarorynbassar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,23 @@
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
```
Где @Table аннотация ?

onggarorynbassar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,23 @@
+    @Column(name = "id", nullable = false)
+    private UUID id;
+    @Column(name = "steam_id", nullable = false)
+    private String steam_id;
```
steamId

onggarorynbassar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,23 @@
+    @Column(name = "steam_id", nullable = false)
+    private String steam_id;
+    @Column(name = "simple_mmr", nullable = false)
+    private int simpleMmr;
+    @Column(name = "competitive_mmr", nullable = false)
+    private int competitiveMmr;
+    @Column(name = "relative_mmr", nullable = false)
+    private int relativeMmr;
```
зачем  указывать  @Column(name = "relative_mmr", nullable = false)? 
Hibernate сам сможет понять что relative_mmr это relativeMmr например. + nullable = false для INT ничего не значит

adikmain committed:
Merge branch 'main' into entities

adikmain committed:
[FIX] Pull request comments

[CHANGE] steam_id -> steamId
[ADD] @Table annotation

onggarorynbassar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,21 @@
+import java.util.UUID;
+
+@Entity
+@Table(name = "player")
```
укажи схему

onggarorynbassar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,21 @@
+    private UUID id;
+    @Column(name = "steam_id", nullable = false)
+    private String steamId;
+    @Column
```
@Column аннотация вообще не нужна

adikmain committed:
[FIX] Pull request comments

[DEL] @Column annotation
[ADD] @Table annotation schema

onggarorynbassar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,21 @@
+import java.util.UUID;
+
+@Entity
+@Table(name = "player", schema = "collector")
```
schema и потом table name

onggarorynbassar commented on src/main/java/kz/onggar/collector/entity/Player.java:
```diff
@@ -0,0 +1,21 @@
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    private String steamId;
```
зря ты убрал @Column nullable false с этой переменной

adikmain committed:
[FIX] Pull request comments

[ADD] @Column annotation over string variable
[FIX] Schema and name order in @Table annotation
